grafana-plugin/src/components/Policy/EscalationPolicy.tsx

```diff
@@ -321,7 +327,11 @@ class _EscalationPolicy extends React.Component<EscalationPolicyProps, any> {
   }
 
   private _renderNotifySchedule() {
```

Comment:
seems we don't need `private` annotation and `_` prefix since we don't use such elsewhere